django_rq/contrib/prometheus.py

```diff
@@ -49,11 +50,18 @@ def collect(self):
                     rq_jobs.add_metric([queue_name, JobStatus.DEFERRED], queue.deferred_job_registry.count)
                     rq_jobs.add_metric([queue_name, JobStatus.SCHEDULED], queue.scheduled_job_registry.count)
 
+                schedulers_statistics = get_scheduler_statistics()
+                rq_scheduled_jobs = GaugeMetricFamily('schedulers_jobs', 'RQ scheduled jobs by schedulers', labels=['sceduler'])
+                if schedulers_statistics.get('schedulers'):
+                    for scheduler, scheduler_stat in schedulers_statistics['schedulers'].items():
```

Comment:
This is always defined so you can just iterate over it directly:

```suggestion
                for key, scheduler_stats in get_schedulers_statistics()['schedulers'].items():
```